dev/pong.py

```diff
@@ -0,0 +1,52 @@
+import lvgl as lv
+from widgets import *
+import app
+import game as g
+import time as t
+
+
+class Ball(g.Sprite):
+
+    # Call the parent class constructor
+    def __init__(self):
+        super().__init__("ball", 3, 3, "BALL")
+        self.direction = [0, 0]
+        self.speed = 0
+        self.reset()
+
+    def reset(self):
+        self.speed = 8.0
+        # make this random
```

Comment:
Could the comment be removed (make an issue for it if necessary), or randomness be introduced?

Reply:
I'll make the rest of the fixes and assign this to @Andrew-Ritchie or @C-Begley as they were working on this